ee/experimental/incident_utils.py

```diff
@@ -66,19 +70,24 @@ def calculate_pmi_matrix(
     
     if not stride:
         stride = os.environ.get('PMI_STRIDE', 60 * 60)
+        
+    if not general_temp_dir:
+        general_temp_dir = os.environ.get('PMI_TEMP_DIR', DEFAULT_TEMP_DIR_LOCATION)
```

Comment:
I don't want infra to be aware of algorithm implementation details, maybe "AI_TEMP_FOLDER"?